### PR TITLE
feat: transform string data for integral fields

### PR DESCRIPTION
**Pull request by dodumosu**
this PR seeks to mitigate app crashes where empty strings are stored for integral fields by converting them

### CODE REVIEW

File: apollo/submissions/utils.py [hunk 1]
```diff
@@ -101,34 +117,34 @@ def make_submission_dataframe(query, form, selected_tags=None,
...
+#     submissions = participant.submissions
+
+#     numerator = 0
+#     denominator = 0
+#     completion_map = {
+#         'Missing': 0,
+#         'Partial': 1,
+#         'Complete': 2,
+#         'Conflict': 2  # TODO: find a better way to compute the ratings
+#     }
```
Comment (takinbo) [inline apollo/submissions/utils.py]: Why are we commenting this out? Isn't it going to be used elsewhere?

Comment (dodumosu) [reply apollo/submissions/utils.py]: we haven't used participant completion rating in years.

Comment (takinbo) [reply apollo/submissions/utils.py]: If it's not been used and there are no references to it, it should be removed and not just commented out.

Comment (takinbo) [reply apollo/submissions/utils.py]: The following references should also be deleted:

https://github.com/nditech/apollo/blob/2192c76af81b6f142204e6a00492e81b0482fa4e/apollo/formsframework/forms.py#L336
https://github.com/nditech/apollo/blob/2192c76af81b6f142204e6a00492e81b0482fa4e/apollo/submissions/views_submissions.py#L1129

**Review by takinbo — APPROVED** [review]

**Pull request merged**